spec/humanize_spec.rb

```diff
@@ -73,6 +73,18 @@
       expect(0.42.humanize(:decimals_as => :digits)).to eql('zero point four two')
     end
 
+    describe 'when set as number' do
+
+      before do
+        Humanize.config.decimals_as = :number
+      end
+
+      it 'reads the leading zeros' do
+        expect(0.0042.humanize).to eql('zero point zero zero forty-two')
```

Comment:
I think this should be zero point zero zero four two. It’s common for numbers after the decimal point to be read individually.

Reply:
Just to clarify. If the decimals_as is set to `:number` and when numbers after decimal point has leading zeros, it will output just like the output when the decimals_as is set as `:digits`?